Scripts/General/config_manager.gd

```diff
@@ -99,6 +99,10 @@ const DEFAULT_ARMY_FORM = preload("res://Scenes/Form/ArmyForm.tscn")
 ## that returns address as a single text line in the response body
 const FETCH_EXTERNAL_IP_GET_URL = "https://api.ipify.org"
 
+const POLYAPI_BASE_URL = "https://polyserver.onrender.com/"
```

Comment:
this has to be configurable

+ I think it would be good as array
+ maybe separate for fetching and broadcastng???